packages/@react-spectrum/combobox/test/ComboBox.test.js

```diff
@@ -3042,6 +3061,85 @@ describe('ComboBox', function () {
           expect(() => getByRole('listbox')).toThrow();
         });
       });
+
+      it('should clear selection and open the menu', () => {
+        let {getByRole} = render(<Component />);
+        let combobox = getByRole('combobox');
+        let button = getByRole('button');
+        act(() => {
+          triggerPress(button);
+          jest.runAllTimers();
+        });
+
+        let listbox = getByRole('listbox');
+        expect(listbox).toBeVisible();
+        let items = within(listbox).getAllByRole('option');
+        expect(items.length).toBe(3);
+
+        act(() => {
+          triggerPress(items[0]);
+          jest.runAllTimers();
+        });
+
+        expect(combobox.value).toBe('One');
+        expect(() => getByRole('listbox')).toThrow();
+
+        // TODO: Figure out why all controlled combobox breaks with a "Can't perform a React state update on an unmounted component"
```

Comment:
Not sure why the below breaks for the all combobox, still trying to debug that one